main/zoe/guide/contracts/treasury.md

```diff
@@ -0,0 +1,147 @@
+# Treasury contract
+
+The Treasury is the primary mechansim for making `RUN` (the Agoric stable-value
+currency) available to participants in the economy. It does this by issuing
+loans against supported types of collateral. The creator of the contract can
+add new types of collateral. (This is expected to be under the control of
+on-chain governance after the initial currencies are defined when the contract
+starts up.)
+
+## Borrowers
+
+Borrowers open a **vault** by calling `makeLoanInvitation()`in the publicAPI to
+get an invitation. Their proposal specifies that they're giving a recognized
+collateral type, and how much `RUN` they want in return. The contract is
+parameterized with a collateralization rate per currency and borrowers can
+withdraw up to that ratio. Other parameters control the interest rate. (Interest
+will be automatically added to the vault's debt.) The contract also has
+access to a priceAuthority, which is an oracle monitoring the exchange value of
+the collateral. If the value of a borrower's collateral ever falls below the
+minimum ratio, the vault will be liquidated. The liquidation approach is
+pluggable and will be modifiable under the control of governance.
+
+When borrowers exercise an invitation and deposit collateral, they receive a
+`Vault` object and some tools useful to the wallet from the offerResults. The
+`Vault` has these methods: `{ makeAdjustBalancesInvitation, makeCloseInvitation,
+getCollateralAmount, getDebtAmount, getLiquidationSeat, getLiquidationPromise
+}`.
+
+An `AdjustBalancesInvitation` allows the borrower to add or remove collateral or
+increase or decrease the loan balance.  `CloseInvitation` allows the borrower
+to close their loan and withdraw any remaining collateral.  The
+`liquidationPromise` and `liquidationSeat` allow the borrower to find out
```

Comment:
Should this part be updated for the recent change?

Reply:
This part of the design ended up not being changed. The discussion we had today included some future changes, but those won't be added immediately.

Reply:
Hmm, is that right? The user still gets a payout through a liquidation promise that they hold? We should never have that.

Reply:
I clarified that the `liquidationPromise` notifies the user about when liquidation happens, and the `liquidationSeat` allows them to collect the proceeds.

Reply:
Ok, sounds good!